### PR TITLE
Test files cleanup

### PR DESCRIPTION
**Pull request by beteva**
Run black and pylint for all the file in test/pytest directory.
Apply python3 syntax.

### CODE REVIEW

File: test/pytest/test_mach_info.py [hunk 1]
```diff
@@ -15,38 +15,44 @@
 #
 # You should have received a copy of the GNU Lesser General Public License
 # along with MXCuBE.  If not, see <https://www.gnu.org/licenses/>.
+"""Test suite for MacuneInfo hardware object
```
Comment (rhfogh) [inline test/pytest/test_mach_info.py:18]: Typo?

File: test/pytest/test_mach_info.py [hunk 2]
```diff
@@ -15,38 +15,44 @@
...
 from test.pytest import TestHardwareObjectBase
+import pytest
 
 __copyright__ = """ Copyright © 2020 by MXCuBE Collaboration """
 __license__ = "LGPLv3+"
 
 
 @pytest.fixture
 def test_object(beamline):
+    """Use the machie_info object from beamline"""
```
Comment (rhfogh) [inline test/pytest/test_mach_info.py:30]: Typo?

File: test/pytest/test_mach_info.py [hunk 3]
```diff
@@ -15,38 +15,44 @@
...
     result = beamline.machine_info
     yield result
     # Cleanup code here - restores starting state for next call:
     # NBNB TODO
 
 
 class TestMachineInfo(TestHardwareObjectBase.TestHardwareObjectBase):
+    """MachineInfo inherots from HardwareObject"""
```
Comment (rhfogh) [inline test/pytest/test_mach_info.py:38]: Typo? 

Does this help - or is the linter enforcing a comment regardless?

Comment (beteva) [reply test/pytest/test_mach_info.py:38]: Thanks @rhfogh for looking at this. Indeed there baere some typos, which I am going to correct ASAP.
Also the linter requires comments, so I've tried to add some, even if it is obvious what the class or the method is doing

Comment (beteva) [reply test/pytest/test_mach_info.py:38]: All good will for more clean-up is appreciated.

**Comment by marcus-oscarsson** [comment]
Well done, thanks !

**Pull request merged**